### ISSUE TITLE
SystemChrome.setPreferredOrientations issues on Flutter 3.7 not fixed in version 3.7.6 as promised

### ISSUE THREAD

**Issue by SangND2000** [flutter/flutter#122574]
```
@override
  void initState() {
    WidgetsBinding.instance.addPostFrameCallback((_) async {
      await SystemChrome.setPreferredOrientations(
        DeviceOrientation.values,
      );
    });
    super.initState();
  }
```

  
it doesn't work again when going back to this screen
it only works once when building the app

**Comment by exaby73** [comment]
Hello @SangND2000. Could you reference the issue where this was promised?

**Comment by SangND2000** [comment]
at version 3.7 in ios seems
```
 WidgetsBinding.instance.addPostFrameCallback((_) async {
      await SystemChrome.setPreferredOrientations(
        DeviceOrientation.values,
      );
    });
```
in setState it only works once
when i went back to the main page and back to the page i needed it didn't work
I read that this bug will be fixed in version 3.7.6 but I updated it and there is still no sign of it being fixed

**Comment by exaby73** [comment]
Where did you read this? I need a reference to the issue in order to understand the issue here
